bindings/java/src/org/sleuthkit/datamodel/AbstractFile.java

```diff
@@ -1315,6 +1325,52 @@ public void save() throws TskCoreException {
 		}
 	}
 
+	/**
+	 * Get the uidStr.
+	 * 
+	 * @return String uid. Returns an empty string if no uid is recorded.
+	 */
+	public String getUidStr() {
```

Comment:
The reason we are using the term uidStr is because we already have a numeric UID & GID that we store in files table.  There is already a method "int getUid()" in AbstractFile. The "str" suffix was to distinguish between the two.

This uidStr field/column is store the long form string for SIDs.  

We kicked around renaming it to ownerUid (still could be confused with the numeric uid)  or ownerUidString

Reply:
Method renamed to getOwnerUid()